oak_functions/sdk/test_utils/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Test utilities to help with unit testing of Oak-Functions SDK code.
+
+use anyhow::Context;
+use log::info;
+
+use std::{path::PathBuf, process::Command};
+
+/// Uses cargo to compile a Rust manifest to Wasm bytes.
```

Comment:
Is there any specific oak functions logic here? AFAICT it is identical to the existing one?

Reply:
Yes I think it would be good, I'm sure I would definitely forget to update one or the other if I made any changes :)